src/NServiceBus.AcceptanceTests/Timeout/CyclingOutageTimeoutPersister.cs

```diff
@@ -95,7 +95,7 @@ public Task<TimeoutData> Peek(string timeoutId, ContextBag context)
             return Task.FromResult<TimeoutData>(null);
         }
 
-        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice)
+        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice, int maxChunkSize = Int32.MaxValue, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Not convinced, the token will be very important to allow the TM to enforce what ever SLA it want to keep so defaulting it sends the wrong message IMO. To me its mandatory to pass it in and make sure to cancel it appropriately. 

Also isn't default on public apis a bad idea? (https://lostechies.com/jimmybogard/2010/05/18/caveats-of-c-4-0-optional-parameters/)